.env.example

```
@@ -4,7 +4,13 @@
+# API key for AI provider
 AI_PROVIDER_API_KEY=your-api-key-here
-# Base URL for AI provider (optional, uses provider default if not set)
-AI_PROVIDER_BASE_URL=
+# Base URL for AI provider
+AI_PROVIDER_BASE_URL=your-provider-base-url-here
+# Model name or identifier for the AI provider (e.g. gpt-4o-mini, claude-2)
+AI_PROVIDER_MODEL=your-model-name-here
+
+# Neon Auth base URL (used by Neon Auth SDK / Data API)
+# Example: https://auth.neon.tech or the value shown in the Neon Console
+AUTH_URL=your-neon-auth-base-url-here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)